datetime.go

```diff
@@ -283,3 +283,230 @@ func (dt *DateTime) NotInRange(min, max time.Time) *DateTime {
 
 	return dt
 }
+
+// Zone returns a new String instance with datetime zone.
+//
+// Example:
+//
+//	tm, _ := time.Parse(time.UnixDate, "FRI Dec 30 15:04:05 IST 2022")
```

Comment:
Could we replace "FRI" with "Fri" in examples? Usually it's how days are spelled.